integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/tests/ServletApiTestApp.java

```diff
@@ -48,25 +48,27 @@ public void test(AbstractServletContainerIntegrationTest test) throws Exception
     }
 
     private void testCaptureBody(AbstractServletContainerIntegrationTest test) throws Exception {
-        test.clearMockServerLog();
-        final Response response = test.getHttpClient().newCall(new Request.Builder()
-            .post(RequestBody.create(MediaType.parse("text/plain"), "{foo}"))
-            .url(test.getBaseUrl() + "/simple-webapp/echo")
-            .build())
-            .execute();
-        assertThat(response.code()).isEqualTo(200);
-        assertThat(response.body().string()).isEqualTo("{foo}");
-
-        final JsonNode transaction = test.assertTransactionReported("/simple-webapp/echo", 200);
-        assertThat(transaction.get("context").get("request").get("body").textValue()).isEqualTo("{foo}");
+        for (String readMethod : List.of(/*"read-byte", "read-bytes",*/ "read-offset", "read-line")) {
```

Comment:
`read-byte` and `read-bytes` are not tested then?

Reply:
another leftover 🤦‍♂️